### PR TITLE
Dashboard widgets filter issue

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**
Run the following command:
php artisan migrate
php artisan db:seed --class="Database\Seeders\v4_0_2\DatabaseSeeder"

### CODE REVIEW

**Review by sandesh556 — APPROVED** [review]

File: app/Http/Controllers/Order/OrderSearchController.php [hunk 1]
```diff
@@ -150,6 +151,7 @@ private function getSelectedVersionOrders($baseQuery, $version, $productId, $req
      */
     public function allInstallations($allInstallation, $join)
     {
+        // dd($allInstallation,$join);
```
Comment (RafficMohammed) [inline app/Http/Controllers/Order/OrderSearchController.php:154]: remove this line

**Review by RafficMohammed — CHANGES_REQUESTED** [review]
Write test cases for this scenario. If we ignore test cases on Billing, Billing will never reach stablity.

**Review by RafficMohammed — DISMISSED** [review]

**Review by AmanpreetLWS — CHANGES_REQUESTED** [review]
First round testing summary:
I tested this PR on existing Database. Found some issues in filters and records shown on pressing view all button . 
Testing time : 6 hours
Total number of issues: 5
Major: 0
Minor: 5
Critical: 0
Is the pr breaking any functionalities? : NO
Number of times went for functionality correction: 1
Issues I found : https://docs.google.com/spreadsheets/d/1ntUnc3OIRd71Q_VilQ9lWtU1hZ1ICoFmafBPcXp8FVU/edit?usp=sharing

Note : 
Found issues not related to pr as well will discuss with team and create separate issues if not existing .

**Comment by AmanpreetLWS** [comment]
First round testing summary:
I tested this PR on existing Database. Above Mentioned Issues where fixed.
Testing time : 2 hours
Total number of issues: 0
Major: 0
Minor: 0
Critical: 0
Is the pr breaking any functionalities? : NO
Number of times went for functionality correction: 1
Issues I found : https://docs.google.com/spreadsheets/d/1ntUnc3OIRd71Q_VilQ9lWtU1hZ1ICoFmafBPcXp8FVU/edit?usp=sharing
Note : 
1. Dashboard now shows same data as in view all (except for module : Products Sold (Last 30 Days) , Total Sold Products)
2. Data for Module (Paid Orders Expiring in Next 30 days) will be shown on the basis of day in month so generally "Feb" has 28 day so the data fill be shown for next 28 day . Data for 29 days expiry not shown .

Screen Shots : 
Products Sold (Last 30 Days) , Total Sold Products :-
Screenshot from 2024-02-19 16-53-18
Data with 29 days of expiry :- 
image
Screenshot from 2024-02-19 16-58-28

**Review by AmanpreetLWS — COMMENTED** [review]
Review Commented Above

**Review by AmanpreetLWS — APPROVED** [review]
All Queries Cleared

**Review by SarfrazS — APPROVED** [review]
Working fine. now whatever we are showing data on dashboard when we click on view all showing same data.
Note:- Product sold by last 30days and Total sold products in this two section when we click on view all show all data there is no filter. For that we have separate issue.

**Review by sandesh556 — APPROVED** [review]

**Pull request merged**